great-idea/css/index.css

```diff
@@ -63,4 +65,207 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/*css for nav bar */
+nav {
+	display: flex;
+	padding-top: 30px;
+	padding-bottom: 20px;
+	width: 100%;
+	margin: 0 auto;
+	width: 100%;
+	align-items: center;
+}
+
+.menu {
+	width: 65%;
+	display: flex;
+	justify-content: space-between;
+}
+
+.menu a {
+	text-decoration: none;
+	color: #cfcfd4;
+	transition: ease-in-out 300ms;
+}
+
+.menu a:hover {
+	color: grey;
+}
+
+.menu-logo {
+	width: 25%;
+	display: flex;
+justify-content: flex-end;
+}
+
+/* style for header */
+.intro-home {
+	display: flex;
+	width: 100%;
+	margin-top: 50px;
+	padding-bottom: 60px;
+}
+
+div.intro {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	font-size: 80px;
+}
+.intro-img {
+	width: 50%;
+	display: flex;
+	justify-content: center;
+}
+
+button {
+	height: 35px;
+	width: 150px;
+	border: 1px solid black;
+	font-size: 16px;
+}
+
+/* style for section */
+.box {
+	display: flex;
+	width: 97%;
+	justify-content: space-between;
+	margin: 0 auto;
+}
+.box-one {
+	width: 48%;
+	margin-bottom: 20px;
+	padding-top: 30px;
+	padding-right: 2px;
+}
+
+.middle-img {
+	display: flex;
+	justify-content: center;
+	margin-top: 10px;
+	width: 98%;
+	padding-left: 2px;
+}
+
+.box-two {
+	width: 30%;
+	padding-top: 33px;
+	padding-bottom: 30px;
+}
+.box-two p{
+	padding-right: 5px;
+}
+/* style for address */
+.address {
+	margin: 0 auto;
+	width: 97%;
+	padding-top: 20px;
+}
+
+address a {
+	padding-top: 20px;
+	text-decoration: none;
+	color: black;
+}
+
+address p:first-child {
+	padding-bottom: 20px;
+}
+
+address p:last-child {
+	padding-top: 20px;
+}
+
+footer {
+	padding-top: 30px;
+	display: flex;
+	justify-content: center;
+	padding-bottom: 20px;
+}
+
+hr {
+	width: 96%;
+	margin-left: 14px;
+}
+
+/* style for service page */
+.container {
+	max-width: 900px;
+}
```

Comment:
I'm glad that you wrapped your pages inside a container div, you still miss a small peace to the puzzle though.  
You probably would like to add `margin: auto` there and you'll see your page will be centered correctly now.